applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -379,7 +380,8 @@ static int initialize(le_audio_receive_cb recv_cb)
 	ARG_UNUSED(recv_cb);
 	if (!initialized) {
 		audio_stream.ops = &stream_ops;
-		ret = bt_audio_unicast_group_create(&audio_stream, CONFIG_BT_ISO_MAX_CHAN,
+		streams_p[0] = &audio_stream;
```

Comment:
This works, but must be improved at some point. Only supports one stream now.
It is kind of hacky to do streams_p[0] when streams_p is an array with the size of CONFIG_BT_ISO_MAX_CHAN.

Reply:
Could we be using streams_p[STREAM_LEFT] = &audio_stream_left, where STREAM_LEFT is a define or enum = 0?